Orange/widgets/visualize/owscatterplotgraph.py

```diff
@@ -766,17 +769,24 @@ def __call__(self):
                         size = size_data
                     widget.scatterplot_item.setSize(size)
                     widget.scatterplot_item_sel.setSize(size + SELECTION_WIDTH)
+                    if widget.timer is None:
+                        widget.end_resizing.emit()
+                    else:
+                        widget.step_resizing.emit()
 
             if self.n_valid <= MAX_N_VALID_SIZE_ANIMATE and \
                     np.all(current_size_data > 0) and np.any(diff != 0):
                 # If encountered any strange behaviour when updating sizes,
                 # implement it with threads
+                self.begin_resizing.emit()
                 self.timer = QTimer(self.scatterplot_item, interval=50)
                 self.timer.timeout.connect(Timeout())
                 self.timer.start()
             else:
+                self.begin_resizing.emit()
```

Comment:
I know that this line repeats and could be before `if`, but I prefer having pairs begin-end.